Sources/pointfreeco/main.swift

```diff
@@ -4,6 +4,11 @@ import Kitura
 import PointFree
 import Prelude
 
+_ = try! migrate()
```

Comment:
This'll crash hard if you don't have a db set up locally. Not sure if we want to add some kind of db setup script (like Rails provides).